### PR TITLE
Shortcodes

### PR DESCRIPTION
**Pull request by Noilleris**
Here is 1 string commit;)

I've added line of code to init shortcodes facade.
I guess in future we should move shortcodes facade to main plugin.

### CODE REVIEW

File: src/class.ziprecipes.php [hunk 1]
```diff
@@ -69,10 +69,14 @@ public static function init()
 
 		closedir($pluginsDirHandle);
 
+        // time to init shortcode facade
+        $shortcodes = new __shortcode();
```
Comment (hgezim) [inline src/class.ziprecipes.php:74]: Where's `__shortcode` required or defined?

File: src/class.ziprecipes.php [hunk 2]
```diff
@@ -395,37 +399,39 @@ public static function zrdn_menu_pages() {
 	}
 
     public static function preload_check_registered() {
-        if ($_SERVER['REQUEST_METHOD'] === 'POST') {
-            foreach ($_POST as $key => $val) {
-                $_POST[$key] = stripslashes($val);
-            }
+        if (isset($_POST['page']) and strpos($_POST['page'], 'zrdn-') !== false) {
```
Comment (hgezim) [inline src/class.ziprecipes.php:403]: Registration changes should not be in this branch.

**Review by hgezim — CHANGES_REQUESTED** [review]

File: src/class.ziprecipes.php (hunk 1 above)
Comment (Noilleris) [reply src/class.ziprecipes.php:74]: nowhere. and i do not quite understand why it so. mybe it is because of namespace.
need to discuss

Comment (Noilleris) [reply src/class.ziprecipes.php:74]: fixed

**Comment by hgezim** [comment]
Merged in #55

**Pull request closed without merging**